### ISSUE TITLE
nxos-bootstrap backtrace when NXOS fails to respond

### ISSUE THREAD

**Issue by jeremyschulman** [Apstra/aeon-ztps#126]
When nxos-bootstrap is in the "AWAIT-ONLINE" state, and the device never comes online, the nxos-bootrap program raises:

````python
2016-12-05 14:37:32,384:ERROR:192.168.159.10:stderr=Traceback (most recent call last):
  File "/opt/aeonztps/bin/nxos-bootstrap", line 403, in <module>
    main()
  File "/opt/aeonztps/bin/nxos-bootstrap", line 392, in main
    dev = wait_for_device(countdown=g_cli_args.reload_delay, poll_delay=10)
  File "/opt/aeonztps/bin/nxos-bootstrap", line 161, in wait_for_device
    message='Failed to probe target %s within reload countdown' % target))
  File "/opt/aeonztps/bin/nxos-bootstrap", line 116, in exit_results
    post_device_status(dev=dev, target=target, state='FAILED', message=results['message'])
  File "/opt/aeonztps/bin/nxos-bootstrap", line 217, in post_device_status
    ip_addr=target or dev.target,
AttributeError: 'NoneType' object has no attribute 'target'
````

The bug here on line 217 is to not do this:
````python
dev.target
````

**Comment by acarpi** [comment]
I have a simple fix for this issue, but the root cause is that sometimes it fails in the do_push_config1 function for this reason

2017-02-05 09:17:03,066:INFO:172.20.232.5:pushing all config to device
2017-02-05 09:17:07,857:CRITICAL:172.20.232.5:unable to push config: % Invalid command

the 'all.conf' config has simply 'service unsupported-transceiver'.

**Comment by acarpi** [comment]
Submitted the fix in PR#153
